libnemo-private/nemo-global-preferences.h

```diff
@@ -214,7 +214,9 @@ typedef enum
 #define NEMO_PREFERENCES_LOCKDOWN_COMMAND_LINE         "disable-command-line"
 
 /* Desktop background */
-#define NEMO_PREFERENCES_SHOW_DESKTOP		   "show-desktop-icons"
+#define NEMO_PREFERENCES_DESKTOP_LAYOUT "desktop-layout"
+#define NEMO_PREFERENCES_SHOW_ORPHANED_DESKTOP_ICONS "show-orphaned-desktop-icons"
+#define NEMO_PREFERENCES_SHOW_DESKTOP   "show-desktop-icons"    /* DEPRECATED */
```

Comment:
This is temporary, I plan on restoring this before 3.0 (as well as add some detection to not try to manage the desktop if something else already is) - it's been noted in other reports already.